src/schemes/fluid/weakly_compressible_sph/rhs.jl

```diff
@@ -25,9 +28,9 @@ function interact!(dv, v_particle_system, u_particle_system,
         rho_mean = 0.5 * (rho_a + rho_b)
 
         # Determine correction values
-        viscosity_correction, pressure_correction = free_surface_correction(correction,
-                                                                            particle_system,
-                                                                            rho_mean)
+        viscosity_correction, pressure_correction, surface_tension_correction = free_surface_correction(correction,
+                                                                                                        particle_system,
+                                                                                                        rho_mean)
```

Comment:
```suggestion
        corrections = free_surface_correction(correction, particle_system, rho_mean)
```
Might be more readable.
And then refer to the correcions:
```julia

        corrections[1] = correction_viscosity
        corrections[2] = correction_pressure
        corrections[3] = correction_surface_tension 
```

Reply:
The current version is more Julian.